app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeUpdate.kt

```diff
@@ -10,7 +10,8 @@ class ConfirmFacilityChangeUpdate : Update<ConfirmFacilityChangeModel, ConfirmFa
       event: ConfirmFacilityChangeEvent
   ): Next<ConfirmFacilityChangeModel, ConfirmFacilityChangeEffect> {
     return when (event) {
-      FacilityChangeConfirmed -> dispatch(ChangeFacilityEffect)
+      is FacilityChangeConfirmed -> dispatch(ChangeFacilityEffect(event.selectedFacility))
+      FacilityChanged -> TODO()
```

Comment:
This is a `TODO` and not a `noChange`. It will be implemented in the next commits.
Also, if I make it a `noChange()` that would be an incorrect implementation.
Since the sheet is not being called from anywhere yet, it's better to keep it as a `TODO`.